scripts/setup-k8s-master.sh.in

```diff
@@ -140,5 +143,8 @@ then
     "INSERT INTO system.locations (\"localityKey\", \"localityValue\", latitude, longitude) VALUES ('region', '{{Region}}', {{RegionLatitude}}, {{RegionLongitude}})"
 fi
 
-
-
+if [[ "{{Workload}}"  =~ KV ]]
+then
+    kubectl run kv --image=cockroachdb/loadgen-kv -- \
+    /kv -drop=false -concurrency 10 -read-percent 95 -tolerate-errors "postgresql://root@${LB_DNS}:26257?application_name=cockroach&sslmode=disable"
```

Comment:
If you're trying to test the overall functionality of the deployment, this is good. If you'd prefer better performance, I'd replace `${LB_DNS}` with `cockroachdb-public` because it's adding extra network hops.